### PR TITLE
Added landing page

### PR DESCRIPTION
**Pull request by Dnyaneshwari-Sonone**

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -2,11 +2,18 @@
     v-app
         PageHeader
         v-content
+<<<<<<< HEAD
```
Comment (Tanuj22) [inline frontend/src/App.vue]: Resolve the conflicts

File: frontend/src/components/HomeLanding.vue [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<template lang="pug">
+v-app#inspire
```
Comment (Tanuj22) [inline frontend/src/components/HomeLanding.vue]: Remove this. Put a `div` if needed

File: frontend/src/components/HomeLanding.vue [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+
+
+
+
+
+</style>
+<script>
+export default {
+  data: () => ({
```
Comment (Tanuj22) [inline frontend/src/components/HomeLanding.vue]: Remove the entire `data` part

**Review by Tanuj22 — CHANGES_REQUESTED** [review]

**Comment by Tanuj22** [comment]
Also please try not to close a PR

**Comment by Dnyaneshwari-Sonone** [comment]
Please review it once again.

File: frontend/src/App.vue [hunk 4]
```diff
@@ -2,7 +2,6 @@
     v-app
         PageHeader
         v-content
-            v-container(fluid)
```
Comment (Tanuj22) [inline frontend/src/App.vue:5]: Why remove this?

Comment (Dnyaneshwari-Sonone) [reply frontend/src/App.vue:5]: For full coverage of the background image.. default padding of the container  was 12px  hence the background was not covering the entire space..

**Pull request merged**